app/models/author.rb

```diff
@@ -1,9 +1,12 @@
-class Author < ApplicationRecord
-  has_many :comments
-  has_many :likes
-  has_many :posts
-
-  def three_recent_posts
-    posts.last(3)
-  end
-end
+class Author < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :posts
+
+  validates :name, presence: true
+  validates :posts_counter, comparison: { greater_than_or_equal_to: 0 }

```

Comment:
Kindly also add the `only_integer` as a validation so as to accept only integers.